recipe/meta.yaml

```diff
@@ -51,30 +51,31 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
     - arm-variant * {{ arm_variant_type }}  # [aarch64]
-    - sysroot_{{ target_platform }} 2.17    # [linux]
   host:
     - patchelf >=0.12  # [linux]
     - cuda-version {{ cuda_major }}.0
     - libcublas  # [(cuda_compiler_version or "").startswith("12")]
+    - {{ stdlib('c') }}
```

Comment:
This is not needed (it's a requirement only for build env).
```suggestion
```

Reply:
Just to confirm, we previously had a `run_constrained` requirement with `sysroot` (`glibc`).
Does the migration imply we no longer need this?